tests/system/small/test_dataframe.py

```diff
@@ -4657,6 +4658,17 @@ def test_to_gbq_and_create_dataset(session, scalars_df_index, dataset_id_not_cre
     assert not loaded_scalars_df_index.empty
 
 
+def test_to_gbq_table_labels(scalars_df_index):
+    destination_table = "bigframes-dev.bigframes_tests_sys.table_labels"
+    result_table = scalars_df_index.to_gbq(
+        destination_table, labels={"test": "labels"}, if_exists="replace"
+    )
+    client = bigquery.Client()
```

Comment:
`client = scalars_df_index._session.bqclient`. Usually, we don't create a separate BQ client, which is expensive and required additional setup, such as default project, token etc.